maintenance/importConfig.ts

```diff
@@ -51,7 +51,8 @@ const program = yargs
 		default: ["oh"],
 	})
 	.option("ids", {
-		description: "devices ids to download",
+		description:
+			"devices ids to download. In ozw the format is '<manufacturer>-<productId>-<productType>'. Ex: '0x0086-0x0075-0x0004'",
```

Comment:
repeating the hex prefix `0x` seems unergonomically - I'd leave them out.

Reply:
It's just because in this way users can simply copy and paste the index shown in z2m ui

Reply:
I see